tutorials/applications/Bayesian Optimisation for antiferromagnetic state preparation.ipynb

```diff
@@ -22,7 +22,7 @@
    "cell_type": "markdown",
```

Comment:
<p>...associated. Its starting and ending values are fixed such that $\Omega(0)=\Omega(T)=0$, $\delta(0)=\delta_0$ and $\delta(T)=\delta_f$.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>